### ISSUE TITLE
Excel and polish fonts (latin extended)

### ISSUE THREAD

**Issue by StRess691** [JulietaUla/Montserrat#104]
Hello. Using polish letters in excel makes an issue with the font. It doesn't matter if the font is light, regular nor bold. When creating one sentence in one row, using polish letters, we would like to bold some part of text, the rest text is going down or up, like on attached screenshot. Up is a sentence with polish letters, down without.
![montserrat](https://user-images.githubusercontent.com/63842522/79546051-04109e80-8092-11ea-9d7c-2f35fc432c58.png)

**Comment by alerque** [comment]
What version exactly are you using and where did you get it? This looks like Windows Excel, have you tried the same thing in other apps? Also can you copy/paste the text sample you are using here so those of us that can't type Polish can use it for testing?

**Comment by StRess691** [comment]
I downloaded fonts from "google fonts", version 7.2.
Problem exists only in Excel, in Word everything is fine. 
This is an Office for Windows, 365 subscription, version of Excel is 16.0.12527.20260 (64-bit).

Polish sentence: "Jakiś tekst pisany czcionką z ogonkami na górze a po prawej bez ogonkow pogrubiony."

**Comment by alerque** [comment]
I don't think this is a problem with the font itself. This might be a repeat of issue #67 which was specifically the version on Google Fonts being broken. You can try downloading the fonts directly from this project not through Google Fonts. That being said I think the result is going to be that this is a bug in Excel (although it's still possible that even with the bug in Excel a different build of the font will avoid triggering the bug, so still try getting font files directly from here).

**Comment by StRess691** [comment]
I deleted fonts spreviously downloaded from "google fonts", downloaded from git, installed and issue still exist. Now tested only on "regular" type of font.

**Comment by FranchettiSpa** [comment]
Hello ,
I have the same problem with accented letters.  
Vertical alignment, in Excel, for accented word is higher than a not accented word.
I've download google font first  then project font but no luck.
I have tried regular and thin weight and problem remain.
I have also check Excel version 2011, and 365.
With Libreoffice Calc, instead, there is no problem.
Thank you
